GEOS_EMCC_FLAGS.mk

```diff
@@ -22,469 +22,244 @@ GEOS_EMCC_FLAGS += -s EXPORTED_FUNCTIONS="[\
   '_initGEOS',\
```

Comment:
This `_initGEOS` and the bottom `_finishGEOS` lines can be deleted, because those are non-reentrant version.
https://github.com/libgeos/geos/blob/main/capi/geos_c.h.in#L2074-L2095

---

And this may be other feature request, but adding `_free` may be good, because I saw that it is used in both Emscripten and GEOS CAPI docs.
- [Interacting with code — Emscripten 3.1.44-git (dev) documentation #Access memory from JavaScript](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.html#access-memory-from-javascript)
  ```js
  var buf = Module._malloc(myTypedArray.length*myTypedArray.BYTES_PER_ELEMENT);
  Module.HEAPU8.set(myTypedArray, buf);
  Module.ccall('my_function', 'number', ['number'], [buf]);
  Module._free(buf);
  ```
- [C API Programming | GEOS #GEOSGeometry
](https://libgeos.org/usage/c_api/#geosgeometry)
  ```c
  /* Two points in an array */
  size_t npoints = 2;
  GEOSGeometry** points = malloc(sizeof(GEOSGeometry*) * npoints);
  points[0] = GEOSGeom_createPointFromXY(0.0, 0.0);
  points[1] = GEOSGeom_createPointFromXY(0.0, 0.0);
  
  /* takes ownership of the points in the array */
  /* but not the array itself */
  GEOSGeometry* collection = GEOSGeom_createCollection(
      GEOS_MULTIPOINT,  /* collection type */
      points,           /* geometry array */
      npoints);
  
  /* frees collection and contained points */
  GEOSGeom_destroy(collection);
  
  /* frees the containing array */
  free(points);
  ```